app/models/container_group.rb

```diff
@@ -95,4 +95,8 @@ def disconnect_inv
     self.deleted_on = Time.now.utc
     save
   end
+
+  def self.display_name(number = 1)
+    n_('Container Pod', 'Container Pods', number)
```

Comment:
Should this just be "Pod" / "Pods"?

Reply:
Also, note that this term may not apply to other variants in the future, which is why we made the model name generic.